tests/manifests/e2e/e2e.yaml

```diff
@@ -27,7 +27,7 @@ spec:
   serviceAccount: tidb-operator-e2e
   containers:
   - name: tidb-operator-e2e
-    image: ""
```

Comment:
@xiaojingchen 
Not sure why it is empty in your #346 PR, can we give it a default value, then it can be applied by `kubectl`?

Reply:
this's ok. in fact the image will be replaced when e2e ci running.